src/GraphQL/Shim/Bool.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+
+namespace GraphQL
+{
+    /// <summary>
+    /// Adapter to unify usages of bool.Parse(ReadOnlySpan) for netstandard2.0 and netstandard2.1
+    /// </summary>
+    internal static class Bool
+    {
+#if NETSTANDARD2_1
+        public static bool Parse(ReadOnlySpan<char> value) => bool.Parse(value);
+#else
+        // copied from .NET Core sources
+        public static bool Parse(ReadOnlySpan<char> value) =>
+            TryParse(value, out bool result) ? result : throw new FormatException($"String was not recognized as a valid Boolean.");
+
+        public static bool TryParse(ReadOnlySpan<char> value, out bool result)
+        {
+            if (IsTrueStringIgnoreCase(value))
+            {
+                result = true;
+                return true;
+            }
+
+            if (IsFalseStringIgnoreCase(value))
+            {
+                result = false;
+                return true;
+            }
+
+            // Special case: Trim whitespace as well as null characters.
+            value = TrimWhiteSpaceAndNull(value);
```

Comment:
Parser library cannot provide whitespace or null characters within a boolean value.

```cs
            if (token.Value == "true" || token.Value == "false")
            {
                return ParseBooleanValue(token);
            }
```

The 'true' or 'false' string also must be lower case.

Reply:
Done. I was going to do it a little later.